site/_guides/cert-manager.md

```diff
@@ -520,8 +516,83 @@ $ curl https://httpbin.davecheney.com/get
 
 cert-manager currently does not have a way to interact with HTTPProxy objects in order to respond to the HTTP01 challenge correctly.
 (See [#950][10] and [#951][11] for details.)
-cert-manager does this by creating a new, temporary Ingress object that will direct requests from Let's Encrypt to temporary pods called 'solver pods'.
-These pods know how to respond to Let's Encrypt's challenge process for verifying you control the domain you're issuing certificates for.
+
+There are two ways to interact with cert-manager which will allow certificates to be requested automatically with an HTTP01 challenge, using an `ingress shim` or using a cert-manager `Certificate` object.
```

Comment:
I'm wondering if there's really any reason to keep the "dummy Ingress" approach around - is there any reason someone would want to use that instead of creating a `Certificate`?

Reply:
No I don't see a need to keep the Ingress approach, I have been using the Certificate object for some time now with success. Previously there was some disagreement from the team on which to use in the guide, but I can't find that issue with that discussion.

That said, I would drop the Ingress method and just use the Certificate, much of it is duplicated.

Reply:
I just re-tested using the Certificate approach and it worked well. I also don't see any reason to keep the dummy ingress approach around since it just adds an unnecessary layer of indirection.

Reply:
@skriss I just removed the ingress-shim docs from the latest commit.